tests/main/ubuntu-core-upgrade-core/task.yaml

```diff
@@ -26,6 +26,19 @@ execute: |
         grub-editenv list | grep "snap_try_core=ubuntu-core_$(cat nextBoot).snap"
         grub-editenv list | grep 'snap_mode=try'
 
+        echo "Ensure the device is scheduled for auto-reboot"
```

Comment:
Interesting.. wonder if we should have a WAIT_REBOOT spread primitive, so we don't actually reboot but rather wait for it to be rebooted.